docs/docs/en/guide/task/flink.md

```diff
@@ -37,7 +37,7 @@ Flink task type, used to execute Flink programs. For Flink nodes:
 | Parallelism             | Used to set the degree of parallelism for executing Flink tasks.                                                                                                                                                                                                        |
 | Yarn queue              | Used to set the yarn queue, use `default` queue by default.                                                                                                                                                                                                             |
 | Main program parameters | Set the input parameters for the Flink program and support the substitution of custom parameter variables.                                                                                                                                                              |
-| Optional parameters     | Set the flink command options, such as `-D`, `-C`, `-yt`.                                                                                                                                                                                                               |
+| Optional parameters     | Set the flink command options, such as `-D`, `-C`, `-yt`, and support the substitution of custom parameter variables.                                                                                                                                                   |
```

Comment:
Please add some examples of how to use it.

Reply:
Ok，I have added examples.